build/_data/elections/oakland/2016-11-08.json

```diff
@@ -26,10 +26,10 @@
       "amount": 850350.55
     },
     {
-      "slug": "city-council-at-large",
-      "type": "office",
-      "title": "City Council At-Large",
-      "amount": 569233.71
+      "slug": "infrastructure-and-housing-bonds",
```

Comment:
At some point, we have to audit why we have differences in text after certain changes.  If you understand where this item is coming from, maybe explain for me why this changed.

Reply:
I think this is due to the fact that we were including data from multiple summaries over the same period so the amount for this candidate dropped and the infrastructure measure rose to the top of the spending.